pkg/doc/verifiable/credential.go

```diff
@@ -1005,6 +1094,86 @@ func issuerToRaw(issuer Issuer) (json.RawMessage, error) {
 	return json.Marshal(issuer)
 }
 
+// SubjectToBytes converts subject(s) to bytes.
+// A subject can be of a different kind:
+// - string (represents subject id)
+// - map
+// - slice of maps
+// - Subject
+// - slice of Subjects
+// - custom struct
+// - slice of custom structs
+func SubjectToBytes(subject interface{}) ([]byte, error) {
```

Comment:
Does this really need to be exported? I believe `Subject.MarshalJSON()` covers this use case.

Reply:
A subject here could be just of `[]Subject` type (as a default result of credential parsing), but can also be a string, custom structure etc. So this is a useful utility function. But after additional consideration, I decided to hide it.